### PR TITLE
Code refactor

### PR DESCRIPTION
**Pull request by tetogomez**
Code refactor

### CODE REVIEW

File: src/components/common/index.js [hunk 1]
```diff
@@ -1,59 +1,63 @@
...
 
-const DropzoneButton = styled(Button)({
-  width: "236px",
-  height: "36px",
-  border: ".5px solid #efefwf",
-  borderRadius: "2px",
-  marginLeft: "auto",
-  marginRight: "auto",
-});
+// const DropzoneBox = styled(Box)({
```
Comment (leisterfrancisco) [inline src/components/common/index.js:21]: @tetogomez is it needed?

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]
Nice work, just one little observation

File: src/components/common/index.js (hunk 1 above)
Comment (tetogomez) [reply src/components/common/index.js]: nice!

**Review by leisterfrancisco — APPROVED** [review]
Good work

**Pull request merged**